fixieai/agents-ts/src/serve-bin.ts

```diff
@@ -0,0 +1,82 @@
+#! /usr/bin/env node
+
+import fs from 'fs';
+import yaml from 'js-yaml';
+import _ from 'lodash';
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
+import serve, { AgentConfig } from './serve';
+
+const { argv } = yargs(hideBin(process.argv))
```

Comment:
The error handling in this file will probably only be encountered by Fixie employees, not by third party devs. But I tried to make it reasonable anyway, to simplify debugging.